pyanaconda/modules/storage/dasd/discover.py

```diff
@@ -0,0 +1,75 @@
+#
+# Discovery tasks
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+import gi
+gi.require_version("BlockDev", "2.0")
+from gi.repository import BlockDev as blockdev
+
+from pyanaconda.core.regexes import DASD_DEVICE_NUMBER
+from pyanaconda.modules.common.task import Task
+from pyanaconda.modules.common.errors.configuration import DiscoveryError
```

Comment:
The inheritance doesn't work for DBus errors, so we would have to map every exception to a new DBus error. Also, I don't see a point in this, because the storage module obviously returns only the storage related discovery errors.

Reply:
Fixed.